docs/framework/get-started/index.md

```diff
@@ -18,72 +18,78 @@ author: "rpetrusha"
- If you are a developer, you can choose any programming language that supports the .NET Framework to create your application. Because the .NET Framework provides language independence and interoperability, you can interact with other .NET Framework applications and components regardless of the language with which they were developed.
 
- To develop .NET Framework applications or components, do the following:
+If you're a developer, choose any programming language that supports the .NET Framework to create your apps. Because the .NET Framework provides language independence and interoperability, you interact with other .NET Framework apps and components regardless of the language with which they were developed.
+
+To develop .NET Framework apps or components, do the following:
+
+1. If it's not preinstalled on your operating system, install the version of the .NET Framework that your app will target. The most recent production version is the .NET Framework 4.7, which is preinstalled on Windows 10 Creative Update and is available for download on earlier versions of the Windows operating system. For .NET Framework system requirements, see [System Requirements](../../../docs/framework/get-started/system-requirements.md). For information on installing other versions of the .NET Framework, see [Installation Guide](../../../docs/framework/install/guide-for-developers.md). Additional .NET Framework packages are released out of band, which means that they're released on a rolling basis outside of any regular or scheduled release cycle. For information about these packages, see [The .NET Framework and Out-of-Band Releases](../../../docs/framework/get-started/the-net-framework-and-out-of-band-releases.md).
 
-1. If it's not preinstalled on your operating system, install the version of the .NET Framework that your application will target. The most recent production version is the .NET Framework 4.7, which is preinstalled on Windows 10 Creative Update and is available for download on earlier versions of the Windows operating system. For .NET Framework system requirements, see [System Requirements](../../../docs/framework/get-started/system-requirements.md). For information on installing other versions of the .NET Framework, see [Installation Guide](../../../docs/framework/install/guide-for-developers.md). There are additional .NET Framework packages that are released out of band. For information about these packages, see [The .NET Framework and Out-of-Band Releases](../../../docs/framework/get-started/the-net-framework-and-out-of-band-releases.md).
+2. Select the .NET Framework language or languages that you intend to use to develop your apps. A number of languages are available, including Visual Basic, C#, Visual F#, and C++ from Microsoft. (A programming language that allows you to develop apps for the .NET Framework adheres to the [Common Language Infrastructure (CLI) specification](http://go.microsoft.com/fwlink/?LinkId=199862).)
```

Comment:
@rpetrusha 

> language or languages that target the .NET Framework

... but the "languages" don't "target" anything, per se, as they're just languages (e.g., VB could be VB6 or VBA or VB.NET ... all implementations of VB). I still favor the latest version ...

> Select the language or languages supported by the .NET Framework that you intend to use to develop your apps.